### DISCUSSION TITLE
odpowiadam na pytania

### DISCUSSION THREAD

**Discussion by wybran** [ElektronPlus/school-website#12] [Q&A]
hejka w dzisiejszym odcinku odpowiem na wasze pytania

**Comment by mtatko** [comment]
dokąd nocą tupta jeż?

**Reply by wybran** [reply]
do dupy na raki łapać szczupaki

**Reply by mtatko** [reply]
dzięki zamykam

**Reply by wybran** [reply]
Do widzenia kup se trąbkę do pierdzenia

**Comment by bkmac511** [comment]
kiedy powrót sanah do radiowęzła?

**Reply by wybran** [reply]
zawsze tam będzie
<img src="https://cdn.discordapp.com/attachments/956973337488265248/958307022024101888/IMG_1965.jpg"/>

**Reply by mtatko** [reply]
peter singer

**Comment by bkmac511** [comment]
co sądzisz o arlecie W

**Reply by mtatko** [reply]
ksiądz nie pomoże
